### ISSUE TITLE
Update the readme.md 

### ISSUE THREAD

**Issue by m0hit-kumar** [m0hit-kumar/m0hit-kumar.github.io#2]
Points to cover:
- Include documentation
- Add how to contribute steps
- And other required info

**Comment by kanishk6103** [comment]
can i work on it?

**Comment by m0hit-kumar** [comment]
yes, you can @kanishk6103, kindly mention the points you will be including in the documentation

**Comment by kanishk6103** [comment]
> yes, you can @kanishk6103, kindly mention the points you will be including in the documentation

I will add a step by step procedure to contribute to open source so that it goes easy for the people who just started to contribute.

**Comment by Korak-Sengupta** [comment]
Can I too work on it? I will attach You Tube video lectures along with it, about open source contribution.

**Comment by m0hit-kumar** [comment]
You can work on it too , but i will accept only one PR request. I will accept that PR which have more detailed and clear guidelines for the contributors. Are you ok with that ? if you still want to do it kindly acknowledge me and i will assign it to you as well. Or you can work on other issue of the project.

**Comment by Korak-Sengupta** [comment]
Yeah, still wanna do it

**Comment by m0hit-kumar** [comment]
> Yeah, still wanna do it

OK, then i assigning it to you. have a great contributing

**Comment by Korak-Sengupta** [comment]
[open.md](https://github.com/m0hit-kumar/m0hit-kumar.github.io/files/9689705/open.md)

**Comment by Korak-Sengupta** [comment]
please have a look.

**Comment by kanishk6103** [comment]
you haven't merged the PR yet? @m0hit-kumar

**Comment by m0hit-kumar** [comment]
> you haven't merged the PR yet? @m0hit-kumar

There is conflict with your PR resolve it and Update your PR

**Comment by kanishk6103** [comment]
> > you haven't merged the PR yet? @m0hit-kumar
> 
> There is conflict with your PR resolve it and Update your PR

oh alright, i'll fix it

**Comment by kanishk6103** [comment]
> > you haven't merged the PR yet? @m0hit-kumar
> 
> There is conflict with your PR resolve it and Update your PR

Done

**Comment by sujalgarg17** [comment]
Please assign me this issue
